pkg/util/overridemanager/overridemanager_test.go

```diff
@@ -14,6 +19,223 @@ import (
 	"github.com/karmada-io/karmada/test/helper"
 )
 
+func Test_overrideManagerImpl_ApplyOverridePolicies(t *testing.T) {
+	deployment := helper.NewDeployment(metav1.NamespaceDefault, "test1")
+	deployment.Labels = map[string]string{
+		"testLabel": "testLabel",
+	}
+	deployment.Annotations = map[string]string{
+		"testAnnotation": "testAnnotation",
+	}
+	deploymentObj, _ := utilhelper.ToUnstructured(deployment)
+
+	clusterRole := helper.NewClusterRole("test2", nil)
+	clusterRole.Labels = map[string]string{
+		"testLabel": "testLabel",
+	}
+	clusterRole.Annotations = map[string]string{
+		"testAnnotation": "testAnnotation",
+	}
+	clusterRoleObj, _ := utilhelper.ToUnstructured(clusterRole)
+	type fields struct {
+		Client        client.Client
+		EventRecorder record.EventRecorder
+	}
+	type args struct {
+		rawObj      *unstructured.Unstructured
+		clusterName string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantCOP *AppliedOverrides
+		wantOP  *AppliedOverrides
+		wantErr bool
+	}{
+		{
+			name: "test overridePolicies",
+			fields: fields{
+				Client: fake.NewClientBuilder().WithScheme(gclient.NewSchema()).WithObjects(&clusterv1alpha1.Cluster{
+					ObjectMeta: metav1.ObjectMeta{Name: "test1"},
+				},
```

Comment:
Replace this logic with `helper.NewCluster("test1"),` will be ok.

Reply:
ok